messaging/src/main/java/org/axonframework/configuration/MessagingConfigurationDefaults.java

```diff
@@ -112,10 +121,26 @@ private static MessageTypeResolver defaultMessageTypeResolver(Configuration conf
         return new ClassBasedMessageTypeResolver();
     }
 
+    private static UnitOfWorkFactory defaultUnitOfWorkFactory(Configuration config) {
```

Comment:
Nit: please mention the `UnitOfWorkFactory` in the defaults list of this class' class-level JavaDoc.